apm-sniffer/apm-agent-core/src/test/java/org/apache/skywalking/apm/agent/core/logging/core/FileWriterTest.java

```diff
@@ -50,6 +54,27 @@ public void testWriteFile() throws InterruptedException {
         Thread.sleep(10000L);
     }
 
+    @Test
+    public void testDeleteWhenRollover() throws InterruptedException {
```

Comment:
@kezhenxu94 Last time your told me, file OPs in CI are dangerous, which could trigger CI unstable, right?

Reply:
@wuguangkuo This test case could be run concurrency because of multiple CI tasks. I believe this could break/fail randomly. I thin if you want to test file delete, you need a UUID in log file name.

Reply:
> @wuguangkuo This test case could be run concurrency because of multiple CI tasks. I believe this could break/fail randomly. I thin if you want to test file delete, you need a UUID in log file name.

I had add UUID string as part of log dir to avoid concurrent problem